### DISCUSSION TITLE
File Uploads

### DISCUSSION THREAD

**Discussion by alexandremjacques** [unpoly/unpoly#402] [Q&A]
Hi,

I have a use case here to upload multiple files via form. Is there something I have to consider if I'm using Unpoly?

I'll try using Dropzone.js but I think I'll have some issues since it works "outside" of the normal form submit flow: https://gitlab.com/meno/dropzone/-/wikis/combine-normal-form-with-dropzone

Does anyone have experience with this kind of situation?

TIA

**Comment by adam12** [comment]
I haven't done anything with Unpoly and file attachments yet (at least that I can distinctly remember), but I suspect it will _just work_ provided the uploader uses an `input[type=file]` element, as Unpoly serializes the `<form>` into `FormData`.

But definitely report back if it doesn't.
